src/pages/guides/review/index.md

```diff
@@ -35,28 +57,84 @@ By creating a project using the SDK in Adobe Developer Console, you are agreeing
 
 Additionally, your integration must not violate the trademarks and copyrights of Adobe, Inc., or those of any other party. Please adhere to all trademark guidelines as noted here: <https://www.adobe.com/legal/permissions/trademarks.html>
```

Comment:
I'd remove the explicit link:

❌ Please adhere to all trademark guidelines as noted here: <https://www.adobe.com/legal/permissions/trademarks.html>

✅ Please adhere to all trademark guidelines as noted [here](https://www.adobe.com/legal/permissions/trademarks.html).